docs/platforms/dotnet/common/troubleshooting.mdx

```diff
@@ -64,4 +64,19 @@ Additionally, if you were using an async main, associated transactions might not
 These issues have been fully resolved as of version 3.31.0 of the Sentry .NET SDK. To safely use an async main, update your version to the newest release.
 It's no longer required to dispose of the result of calling `SentrySdk.Init`.
 
+## Obfuscated Code and Stack Traces
+
+
+It's important to define what obfuscation means. If you're getting illegible stack traces because of missing [debug files](/platforms/dotnet/data-management/debug-files/), all you need to do is enabling debug file upload, which is done automatically by the [Sentry SDK via MSBuild](/platforms/dotnet/guides/aspnet/configuration/msbuild/).
+
+Some custom tools obfuscates .NET Intermediate Language (IL) in order to make apps harder to reverse-engineer; Sentry does not currently support these tools. This is because each obfuscator uses its own custom logic when dealing with deobfuscation, making it impractical to implement universal support.
```

Comment:
```suggestion
Some custom tools obfuscate .NET Intermediate Language (IL) in order to make apps harder to reverse-engineer; Sentry does not currently support these tools. This is because each obfuscator uses its own custom logic when dealing with deobfuscation, making it impractical to implement universal support.
```